### ISSUE TITLE
Warning: The command you entered did not catch on any hooks when I try to deploy the plugin

### ISSUE THREAD

**Issue by thomasfaller** [nya1/serverless-ignore#5]
I have the following serverless.yml file :

```yml
service: zendesk

frameworkVersion: "2"

provider:
  name: aws
  runtime: python3.8
  stage: "dev"
  region: eu-west-1
  lambdaHashingVersion: 20201221

functions:
  collections-next14days:
    handler: col14d.main
  deliveries-next14days:
    handler: del14d.main
  add-comment:
    handler: add-comment.main
  tag-first-comment:
    handler: tag-first-comment.main
  tag-last-reply:
    handler: tag-last-reply.main
  create-ticket:
    handler: create-ticket.main
  create-ticket-w-attachement:
    handler: create-ticket-w-attach.main
  create-viagra-ticket:
    handler: create-viagra-ticket.main
  create-stripe-order-ticket:
    handler: create-stripe-order-ticket.main

plugins:
  - serverless-python-requirements
  # - serverless-ignore

custom:
  test: true
  stages:
    - dev
    - prod
```

All is working well and I can deploy via `serverless deploy` but as soon as I install serverless-ignore (via npm --save-dev) and add it in the .yml file in my list of plugins I get the following log at deploy :

```bash
Serverless: Deprecation warning: Detected ".env" files. In the next major release variables from ".env" files will be automatically loaded into the serverless build process. Set "useDotenv: true" to adopt that behavior now.
            More Info: https://www.serverless.com/framework/docs/deprecations/#LOAD_VARIABLES_FROM_ENV_FILES
Serverless: Load command interactiveCli
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command config:tabcompletion
Serverless: Load command config:tabcompletion:install
Serverless: Load command config:tabcompletion:uninstall
Serverless: Load command create
Serverless: Load command install
Serverless: Load command package
Serverless: Load command deploy
Serverless: Load command deploy:function
Serverless: Load command deploy:list
Serverless: Load command deploy:list:functions
Serverless: Load command invoke
Serverless: Load command invoke:local
Serverless: Load command info
Serverless: Load command logs
Serverless: Load command metrics
Serverless: Load command print
Serverless: Load command remove
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command slstats
Serverless: Load command plugin
Serverless: Load command plugin
Serverless: Load command plugin:install
Serverless: Load command plugin
Serverless: Load command plugin:uninstall
Serverless: Load command plugin
Serverless: Load command plugin:list
Serverless: Load command plugin
Serverless: Load command plugin:search
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command upgrade
Serverless: Load command uninstall
Serverless: Load command requirements
Serverless: Load command requirements:clean
Serverless: Load command requirements:install
user: thomas zendesk on  master [✘!?] via ⬢ v14.17.0 via 🐍 v3.8.6 (venv) via  13GiB/23GiB | 0B/4GiB on ☁️  (eu-west-1) took 2s 
🕙[ 18:55 ] ➜ SLS_DEBUG=* serverless deploy
Serverless: Deprecation warning: Detected ".env" files. In the next major release variables from ".env" files will be automatically loaded into the serverless build process. Set "useDotenv: true" to adopt that behavior now.
            More Info: https://www.serverless.com/framework/docs/deprecations/#LOAD_VARIABLES_FROM_ENV_FILES
Serverless: Load command interactiveCli
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command config:tabcompletion
Serverless: Load command config:tabcompletion:install
Serverless: Load command config:tabcompletion:uninstall
Serverless: Load command create
Serverless: Load command install
Serverless: Load command package
Serverless: Load command deploy
Serverless: Load command deploy:function
Serverless: Load command deploy:list
Serverless: Load command deploy:list:functions
Serverless: Load command invoke
Serverless: Load command invoke:local
Serverless: Load command info
Serverless: Load command logs
Serverless: Load command metrics
Serverless: Load command print
Serverless: Load command remove
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command slstats
Serverless: Load command plugin
Serverless: Load command plugin
Serverless: Load command plugin:install
Serverless: Load command plugin
Serverless: Load command plugin:uninstall
Serverless: Load command plugin
Serverless: Load command plugin:list
Serverless: Load command plugin
Serverless: Load command plugin:search
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command upgrade
Serverless: Load command uninstall
Serverless: Load command deploy
Serverless: Load command requirements
Serverless: Load command requirements:clean
Serverless: Load command requirements:install
Serverless: Load command requirements:cleanCache
Serverless: Load command login
Serverless: Load command logout
Serverless: Load command generate-event
Serverless: Load command test
Serverless: Load command dashboard
Serverless: Load command output
Serverless: Load command output:get
Serverless: Load command output:list
Serverless: Load command param
Serverless: Load command param:get
Serverless: Load command param:list
Serverless: Load command studio
Serverless: Skipping variables resolution with old resolver (new resolver reported no more variables to resolve)
Serverless: Invoke deploy
Serverless: Warning: The command you entered did not catch on any hooks
```
As soon as I comment out `- serverless-ignore` I can deploy just as expected.

**Comment by mkeemon** [comment]
@thomasfaller I ran into the same issue. I opted to use the `package.patterns` config

https://www.serverless.com/framework/docs/providers/aws/guide/packaging/

**Comment by CORT0619** [comment]
@mkeemon i'm using package.patterns but it's not working for me.
my folder structure is as follows: 
![image](https://user-images.githubusercontent.com/17831460/123110210-dd729400-d409-11eb-9dfc-321fa2aafcd2.png)


 i'm doing: 
``` package:
  patterns:
    - "!src/__tests__/**"
    - "!src/__mocks__/**"
    - "!src/__test__utils/**"

**Comment by nya1** [comment]
fixed in 0.2.1 version
